.gitignore

```diff
@@ -2,3 +2,12 @@
 .dart_tool/
 .packages
 pubspec.lock
+
```

Comment:
Where do these come from?

Reply:
These are the internal diffs - files which exist in google3 but not on github.